src/Build.UnitTests/BackEnd/AssemblyLoadContextTestTasks.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace AssemblyLoadContextTest
+{
+    public class RegisterObject : Task
+    {
+        internal const string CacheKey = "RegressionForMSBuild#5080";
```

Comment:
Do you always have to have CacheKeys like this and register them in Execute when introducing a new task for them to be used in MSBuild code?

Reply:
I don't understand the question. Can you elaborate?

Reply:
I was wondering what extra steps there are to let a task be called from MSBuild without a UsingTask element.